leftwm/src/config/keybind.rs

```diff
@@ -0,0 +1,169 @@
+use super::BaseCommand;
+use crate::Config;
+use anyhow::{ensure, Context, Result};
+use leftwm_core::layouts::Layout;
+use serde::{Deserialize, Serialize};
+use std::str::FromStr;
+
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub struct Keybind {
+    pub command: BaseCommand,
+    #[serde(default)]
+    pub value: String,
+    pub modifier: Option<Modifier>,
```

Comment:
This is fantastic! I wasn't able to get the modifier = None case to work locally, but otherwise this PR looks good to me.